src/analysis/special_functions/trigonometric/chebyshev.lean

```diff
@@ -54,10 +59,23 @@ begin
     simp only [add_mul, one_mul] }
 end
 
-/-- `sin ((n + 1) * θ)` is equal to `sin θ` multiplied with the `n`-th Chebyshev polynomial of the
-second kind evaluated on `cos θ`. -/
-lemma sin_nat_succ_mul (n : ℕ) (θ : ℂ) :
-  sin ((n + 1) * θ) = (U ℂ n).eval (cos θ) * sin θ :=
-(U_complex_cos θ n).symm
+end complex
+
+/- ### Real versions -/
+
+section real
+open real
+
+/-- The `n`-th Chebyshev polynomial of the first kind evaluates on `cos θ` to the
+value `cos (n * θ)`. -/
+@[simp] lemma T_real_cos (θ : ℝ) (n : ℕ) : (T ℝ n).eval (cos θ) = cos (n * θ) :=
+by { rw [←complex.of_real_inj, ←complex.coe_algebra_map, ←aeval_algebra_map_apply], simp }
```

Comment:
It really ought to be possible to use `by exact_mod_cast T_complex_cos θ n` as the proof here. I think the fact that you can't indicates we're missing a `norm_cast` lemma:

```lean
@[simp, norm_cast] lemma of_real_T_eval (x : ℝ) (n : ℕ) : ((T ℝ n).eval x : ℂ) = (T ℂ n).eval x :=
by simp_rw [←complex.of_real_eq_coe, ←eval₂_at_apply, eval₂_eq_eval_map, map_T]
```

(and likewise for `U`, of course).

Reply:
I'm not sure what the best place to put such a `norm_cast` lemma is, however.

Reply:
This is a special case of a more general lemma involving `algebra_map`, which I've also added to the PR.